examples/worlds/lift_drag_nested.sdf

```diff
@@ -0,0 +1,293 @@
+            <ixz>0.0</ixz>
+            <iyy>0.012</iyy>
+            <iyz>0.0</iyz>
+            <izz>0.0012</izz>
+          </inertia>
+          <mass>1.0</mass>
+        </inertial>
+        <collision name="collision">
+          <geometry>
+            <box>
+              <size>0.02 0.02 0.5</size>
+            </box>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <box>
+              <size>0.02 0.02 0.5</size>
+            </box>
+          </geometry>
+          <material>
+            <ambient>0.3 0.3 0.3 1.0</ambient>
+            <diffuse>.421 0.225 0.0 1.0</diffuse>
+          </material>
+        </visual>
+      </link>
+      <model name="blade_1">
+        <link name="link">
+          <pose>0 0 1.5 0.0 0 0</pose>
+          <inertial>
+            <pose>0.0 0.5 0 0.0 0.0 0.0</pose>
+            <inertia>
+              <ixx>0.0000465</ixx>
+              <ixy>0.0</ixy>
+              <ixz>0.0</ixz>
+              <iyy>0.0000006</iyy>
+              <iyz>0.0</iyz>
+              <izz>0.0000470</izz>
+            </inertia>
+            <mass>0.01</mass>
+          </inertial>
+          <collision name="collision">
+            <pose>0.0 0.5 0 0.0 0.0 0.0</pose>
+            <geometry>
+              <box>
+                <size>0.2 1.0 0.01</size>
+              </box>
+            </geometry>
+          </collision>
+          <visual name="visual">
+            <pose>0.0 0.5 0 0.0 0.0 0.0</pose>
+            <geometry>
+              <box>
+                <size>0.2 1.0 0.01</size>
+              </box>
+            </geometry>
+            <material>
+              <ambient>0.5 0.2 0.2 1.0</ambient>
+              <diffuse>.421 0.225 0.0 1.0</diffuse>
+            </material>
+          </visual>
+        </link>
+      </model>
+      <model name="blade_2">
+        <link name="link">
+          <pose>0 0 1.5 0.0 0 3.141593</pose>
+          <inertial>
+            <pose>0.0 0.5 0 0.0 0.0 0.0</pose>
+            <inertia>
+              <ixx>0.0000465</ixx>
+              <ixy>0.0</ixy>
+              <ixz>0.0</ixz>
+              <iyy>0.0000006</iyy>
+              <iyz>0.0</iyz>
+              <izz>0.0000470</izz>
+            </inertia>
+            <mass>0.01</mass>
+          </inertial>
+          <collision name="collision">
+            <pose>0.0 0.5 0 0.0 0.0 0.0</pose>
+            <geometry>
+              <box>
+                <size>0.2 1.0 0.01</size>
+              </box>
+            </geometry>
+          </collision>
+          <visual name="visual">
+            <pose>0.0 0.5 0 0.0 0.0 0</pose>
+            <geometry>
+              <box>
+                <size>0.2 1.0 0.01</size>
+              </box>
+            </geometry>
+            <material>
+              <ambient>0.2 0.5 0.2 1.0</ambient>
+              <diffuse>.421 0.225 0.0 1.0</diffuse>
+            </material>
+          </visual>
+        </link>
+    </model>
```

Comment:
nit: weird indent

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/955/commits/aa26825053a65c20700291b9b988ff12de37704e